### ISSUE TITLE
Be able to add records to data from fluent-bit

### ISSUE THREAD

**Issue by greggilbert** [fluent/fluent-bit#191]
I want to be able to attach multiple tags to data from the fluent-bit side. For example, I want data from `syslog` on my dev server to be tagged as `env=dev` and `logsource=syslog`, and data from `syslog` on my prod server to be tagged as `env=prod` and `logsource=syslog`. By having multiple tags, in Graylog I can search for `env:"dev" AND logsource: "syslog"`.

To accomplish this now, you'd have to do this:

```
# fluent-bit
[INPUT]
    Name tail
    Path /var/log/syslog
    Tag dev_syslog

[INPUT]
    Name tail
    Path /var/log/app.log
    Tag dev_app
```

```
# fluentd
<filter dev_syslog>
  @type record_transformer
  <record>
    env "dev"
    logsource "syslog"
  </record>
</filter>

<filter dev_app>
  @type record_transformer
  <record>
    env "dev"
    logsource "app"
  </record>
</filter>
```

It would be much easier if the fluent-bit side could send data with the records already attached. For example:

```
# hypothetical fluent-bit
[INPUT]
    Name tail
    Path /var/log/syslog
    Env dev
    Logsource syslog

[INPUT]
    Name tail
    Path /var/log/app.log
    Env dev
    Logsource app
```

and then you wouldn't have to do the filter matching within Fluentd.

**Comment by edsiper** [comment]
thanks.

In order to keep compliance with Fluentd, this will be implemented through a Filter, either like _record\_transformer_ or _record\_modifier_.

**Comment by edsiper** [comment]
Already implemented in GIT Master through PR #235 (upcoming into v0.12 release)
